tools/generator/filters/common.py

```diff
@@ -41,3 +59,8 @@ def generic_comment(value: str, start: str, indent: int = 0) -> str:
         break_long_words=False,
         replace_whitespace=False
     ))
+
+
+@register_function
+def array_types(game):
+    return SIMPLE_TYPES + tuple([s['str_name'] for s in game['struct']])
```

Comment:
Maybe we should filter out the array types not used in the game?

For instance, for tictactoe we declare double_array, but there is no double.

Reply:
Yes, probably a good idea.